src/main/frontend/handler/editor.cljs

```diff
@@ -3319,32 +3331,35 @@
     [{:block a :level 1}
      {:block b :level 2}
      {:block e :level 2}]"
-  [{:keys [collapse?] :or {collapse? false}}]
+  [{:keys [collapse? expanded? root-block max-level] :or {collapse? false expanded? false root-block nil max-level nil}}]
   (when-let [page (or (state/get-current-page)
                       (date/today))]
     (->>
      (-> page
          (db/get-page-blocks-no-cache)
          (tree/blocks->vec-tree page))
 
-     (#(if collapse?
-         (w/postwalk
-          (fn [x]
-            (if (and (map? x) (-> x :block/properties :collapsed))
```

Comment:
missing `(map? x)` logic, will not guard x against nil.
`Error: Assert failed: (string? s)`

Reply:
I changed the code to use cond->> for consistence.
```clojure
    (#(cond->> %
         collapse? (w/postwalk
                     (fn [x]
                      (if (and (map? x) (-> x :block/properties :collapsed))
                        (assoc x :block/children []) x)))))
```